datastore-api/app/core/kgs/connector.py

```diff
@@ -285,6 +301,8 @@ async def get_nodes_for_nodepairs(self, kg_name, nid_pairs:Tuple[str, str]):
             results.append(await self.get_nodes_for_nodepair(kg_name, [in_id, out_id]))
 
         return results
+
+
     async def get_edge_msearch(self, kg_name, nids_pairs: List[Tuple[str, str]]):
         """Returns all edges for a given node-pair.
```

Comment:
A mismatch between the input type and the code desc: List of node pairs vs a node pair